packages/ove-service-auth/README.md

```diff
@@ -0,0 +1,191 @@
+**services.json** file.
+- **path** is used in the forwarded request 
+
+#### Global proxy
+
+In **global proxy** mode all requests are forwarded to the provided url. The url is configured using the 
+**SERVICE_GLOBAL_PROXY_URL** environment variable. 
+
+**Example:** The proxy is running on **localhost:8080** and service-abc responds to requests on 
+**localhost:9090/search/<query>**. We would like to proxy all these requests and map **localhost:8080** -> 
+**localhost:9090/search/** therefore the **SERVICE_GLOBAL_PROXY_URL** will be set to **localhost:9090/search/**.
+
+#### Service proxy
+
+In **service proxy** mode, the request are mapped in a similar way, but the mapping rules are described by the 
+**services.json** config file. Example:
+
+```json
+{
+  "ove": [
+    "http://localhost:3004/",
+    "http://localhost:3005/"
+  ]
+}
+```
+
+In this case, the proxy is running on **localhost:8080** and two ove services are running on **localhost:3004** 
+and **localhost:3005**. In case of a request to **locahost:8080/ove/** this will be forwarded to either 
+**localhost:3004** or **localhost:3005** by using a very simple load balancing strategy.
+
+**Note**: This proxy service assumes that all the **state synchronisation** is performed by the services.
+
+#### Important note on proxies
+
+The proxy mapping rules are overridden by the rest service mappings. In case authentication and locking are enabled
+the proxy will serve these routes instead of proxying the requests. Currently all the auth operations are performed 
+under **auth/login** and **auth/lock** routes.
+
+### Authentication
+
+This feature allows users to authenticate with standard username and password. The authentication is performed using 
+a standard jwt protocol. The user config, with username and passwords are described in the **config/users.json** file:
+
+```json
+[
+  {
+    "id": "admin",
+    "password": "secure-password-123"
+  }
+]
+``` 
+
+**Note:** The user id has to be unique.
+
+All the other authentication parameters can be set using environment variables:
+
+- **SERVICE_LOGIN_ENABLED**: enable the login feature, default is True.
+- **SERVICE_LOGIN_CONFIG**: location of the login configuration file, containing the usernames and passwords. The default 
+location is  **config/users.json**.
+- **SERVICE_LOGIN_HASH_PASSWORDS**: enable the password hashing feature after loading from file, default is True. 
+Must be enabled if the passwords are stored in plain text.
+- **JWT_TOKEN**: a secure token to be used by the JWT algorithm. By default a secure random password is generated.
+- **JWT_ISSUER**: the name of the issuer to be used by the JWT algorithm, default is heimdall. This can be personalized 
+to something more secure.
+- **JWT_TOKEN_EXPIRATION**: JWT token expiration parameter (in seconds), default is 3600. 
+
+### Authorization
+
+Authorization can be configured for every route by using a read or write token. These can be configured in 
```

Comment:
If I don't set a read/write token, does that mean that authentication s not required to read/write? If so, say so explicitly.